components/dotenv.rst

```diff
@@ -62,9 +62,52 @@ The ``load()`` method never overwrites existing environment variables. Use the
     // ...
     $dotenv->overload(__DIR__.'/.env');
 
+As you're working with the ``Dotenv`` component you'll notice that you might want
```

Comment:
We can remove the inline code block here, it makes reading a bit easier and strictly spoken we refer to the name of a component and not a code element here.

Reply:
Done.